exec/java-exec/src/main/java/org/apache/drill/exec/store/mock/package-info.java

```diff
@@ -60,14 +62,26 @@
  * <li>The <tt>mode</tt> is one of the supported Drill
  * {@link DataMode} names: usually <tt>OPTIONAL</tt> or <tt>REQUIRED</tt>.</li>
  * </ul>
+ * <p>
+ * Recent extensions include:
+ * <ul>
+ * <li><tt>repeat</tt> in either the "entry" or "record" elements allow
```

Comment:
Yes. Added the property to MockScanEntry. Need to add it to the implementation as well, which is planned, but not yet complete.